chat-plugins/info.js

```diff
@@ -460,6 +288,15 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature." + "\n" +
```

Comment:
I thought it made it easier to read the help for the commands from within the file. If you'd prefer it the other way I can change it.